README.md

```diff
@@ -1,10 +1,11 @@
 # Mobile-Version
 - This is the second part of building my portfolio website
-- I will create sections within the website where i will include my portfolio projects.
+- I will create sections within the website where I will include my portfolio projects.
```

Comment:
- [ ] Please update your readme header following the [Microverse Readme template](https://github.com/microverseinc/readme-template) and descript the different steps necessary for another developer to clone and make use of the project.